releaf-core/app/lib/releaf/settings/normalize_value.rb

```diff
@@ -0,0 +1,45 @@
+class Releaf::Settings::NormalizeValue
+  include Releaf::Service
+  attribute :value, Object
+  attribute :input_type, Symbol
+
+  def call
+    if self.class.respond_to? normalization_method
+      self.class.send(normalization_method, value)
+    else
+      value
+    end
+  end
+
+  def normalization_method
+    "normalize_#{input_type}"
+  end
+
+  def self.normalize_decimal(value)
+    value.to_s.sub(",", ".").to_d
+  end
+
+  def self.normalize_float(value)
+    value.to_s.sub(",", ".").to_f
+  end
+
+  def self.normalize_integer(value)
+    value.to_i
+  end
+
+  def self.normalize_time(value)
+    Time.parse(value)
```

Comment:
```
Time.parse('')
TIme.parse(nil)
Time.parse('foo')
Time.parse(1)
...
```
all will fail with some sort of exception (ArgumentError or TypeError).

Is that intentional?
Same for DateTime and Date.